rotkehlchen/tests/api/test_user_assets.py

```diff
@@ -90,28 +92,35 @@ def test_add_user_assets(rotkehlchen_api_server, globaldb):
     assert data.coingecko == user_asset2['coingecko']
     assert data.cryptocompare == user_asset2['cryptocompare']
 
-    # try to add a token type/name/symbol combo that exists
-    bad_asset = {
-        'asset_type': 'fiat',
-        'name': 'Euro',
-        'symbol': 'EUR',
+    # Add an evm token with the user asset endpoint
+    new_evm_token_address = make_evm_address()
```

Comment:
Shuldn't you also add a test for adding a new fiat asset through this endpoint?

Reply:
There actually already is a test a bit lower. I forgot to rename a variable from `bad_asset`, but otherwise the test is there. 

```python

    # Edit a fiat asset with the user asset endpoint
    bad_asset = {
        'identifier': 'EUR',
        'asset_type': 'fiat',
        'name': 'Edited EUR',
        'symbol': 'NEWEUR',
    }
    response = requests.patch(
        api_url_for(
            rotkehlchen_api_server,
            'allassetsresource',
        ),
        json=bad_asset,
    )
    assert_proper_response(response)
    resolved_eur = A_EUR.resolve_to_fiat_asset()
    assert resolved_eur.name == 'Edited EUR'
    assert resolved_eur.symbol == 'NEWEUR'
```

Reply:
That is editing a fiat asset and it's still called `bad_asset`. I am asking foir **adding** one